data/yaml/page-furniture/sidebar.yaml

```diff
@@ -157,11 +157,32 @@ items:
             link: /protocols/pusher
           - label: 'PubNub Adapter'
             link: /protocols/pubnub
-      - label: 'Ably accounts and apps'
+      - label: 'Accounts and apps'
         link: ''
         items:
-          - label: 'User management'
+          - label: 'Overview'
+            link: /account
+          - label: 'Settings'
+            link: /account/settings
+          - label: 'Users'
```

Comment:
I think this should go back to 'user management' with the latest version. And potentially drop down below 'App management'?